sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/src/AuthenticationEventBinding.cs

```diff
@@ -86,8 +86,13 @@ private static IReadOnlyDictionary<string, Type> GetBindingDataContract(Paramete
         public async Task<ITriggerData> BindAsync(object value, ValueBindingContext context)
         {
             var request = (HttpRequestMessage)value;
+#if NET8_0_OR_GREATER
```

Comment:
[HttpRequestMessage.Properties](https://learn.microsoft.com/en-us/dotnet/api/system.net.http.httprequestmessage.properties?view=net-8.0) is now obsolete. The guidance is to use [HttpRequestMessage.Options](https://learn.microsoft.com/en-us/dotnet/api/system.net.http.httprequestmessage.options?view=net-8.0) instead. However, the Options property is not available in netstandard2.0.